File: Components/BlogML/Xml/BlogMLMeta.vb
Imports System.Xml.Serialization

Namespace BlogML.Xml
 <Serializable, Obsolete("I don't think that we use this now that we are using Dictionary<K,V>")>
 Public NotInheritable Class Meta

  <XmlAttribute("type")>
  Public Property Type As String

  <XmlAttribute("value")>
  Public Property Value As String

 End Class
End Namespace

File: Server/Core/BlogML/Xml/BlogMLMeta.cs
using System;
using System.Xml.Serialization;

namespace DotNetNuke.Modules.Blog.Core.BlogML.Xml
{
  [Serializable]
  [Obsolete("I don't think that we use this now that we are using Dictionary<K,V>")]
  public sealed class Meta
  {

    [XmlAttribute("type")]
    public string Type { get; set; }

    [XmlAttribute("value")]
    public string Value { get; set; }

  }
}